Assets/Thirdweb/Scripts/Utils.cs

```diff
@@ -53,5 +55,37 @@ public static long UnixTimeNowMs()
             var timeSpan = (DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0));
             return (long)timeSpan.TotalMilliseconds;
         }
+
+        public static string ToWei(this string eth)
+        {
+            double ethDouble = 0;
+            if (!double.TryParse(eth, out ethDouble))
+                return "INVALID_ARGUMENTS";
+            BigInteger wei = (BigInteger)(ethDouble * DECIMALS_18);
+            return wei.ToString();
+        }
+
+        public static string ToEth(this string wei)
+        {
+            BigInteger weiBigInt = 0;
+            if (!BigInteger.TryParse(wei, out weiBigInt))
+                return "INVALID_ARGUMENTS";
+            double eth = (double)weiBigInt / DECIMALS_18;
+            return eth.ToString();
```

Comment:
how about just calling `FormatERC20` here? that way we can expose `decimalsToDisplay` as an optional param on `ToEth` too which is super useful

Reply:
Yep good point!